setup.py

```diff
@@ -75,7 +75,7 @@
         "pyyaml",
         "requests",
         "s3fs",
-        "smartystreets-python-sdk >= 4.0.1",
+        "smartystreets-python-sdk <= 4.6.1",
```

Comment:
While unlikely to be an issue in our environment, this isn't the right pin, since it doesn't preserve the minimum version req. I believe `>= 4.0.1, <= 4.6.1` is the combination of the two.

Reply:
I just force pushed. Does that look right now?

Reply:
LGTM. Please test it, of course.